spec.md

```diff
@@ -916,6 +922,7 @@ Service Broker will use it to correlate the resource it creates.
 | organization_guid* | string | Deprecated in favor of `context`. The Platform GUID for the organization under which the Service Instance is to be provisioned. Although most Service Brokers will not use this field, it might be helpful for executing operations on a user's behalf. MUST be a non-empty string. |
 | space_guid* | string | Deprecated in favor of `context`. The identifier for the project space within the Platform organization. Although most Service Brokers will not use this field, it might be helpful for executing operations on a user's behalf. MUST be a non-empty string. |
 | parameters | object | Configuration parameters for the Service Instance. Service Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
+| maintenance_info | object | If a Service Broker provides maintenance information for a Service Plan in its [Catalog](#catalog-management), a Platform MAY provide the same maintenance information when provisioning a Service Instance. This field can be used to ensure that the end-user of a Platform are provisioning what they are expecting since maintenance information can be used to describe important information (such as the version of the operating system the Service Instance will run on). If a Service Broker's catalog has changed and new maintenance information is available for the Service Plan being provisioned, then the Service Broker MUST reject the request with a `422 Unprocessable Entity` as described below. |
```

Comment:
This implies that the end-user has seen and understood the maintenance info. Without semantics, the platform can only present the plain JSON object. I guess many end-users don't understand what they are looking at.
And how should an end-user judge whether the OS version is good or a bad for a service instance? The OS a database is running on, for example, is an implementation detail and should be hidden from an end-user.

Reply:
I don't think this implies that the end user has always completely understood the maintenance info. Service authors can select the info that they think might be useful to end users, but not every end user needs to understand the whole object. 
At the lowest level, a basic user needs to know that there is maintenance available. More experienced users might be interested in the details of the maintenance.

Showing maintenance information allows users to work towards better understanding of the services they use, and allows already accomplished users to make good decisions about applying maintenance operations. Hiding information about maintenance disempowers both new and experienced users.

The platform can create a range of possible user experiences when displaying maintenance info. Examples could include: displaying the whole JSON as you suggest; displaying an itemised diff; displaying only a notification that the maintenance info has changed, and perhaps others. If this does not provide enough flexibility for platforms, the spec could be made more restrictive in the future.

Reply:
Most services provided via OSB are managed services. There is an inherent disempowerment. The only choice the user has is when to apply the update and not if the update should be deployed at all. If the update fixes a security issue, the service instance will be updated anyway automatically at some point.
Eventually, this boils down to a hint that there is an update available with some high-level description. Providing service internals is a bad design, is unnecessary, and in the worst case can be a security issue.

Reply:
I agree there is an inherent disempowerment with managed services. On the one hand they free users of the burden of maintaining their service, but on the other there is a risk that users are not able to use the service effectively.

Providing some information about the internals of a service need not represent bad design. Often some information is useful to make decisions about how a service is used, or to help make judgements about how it will respond. 

Importantly, providing a reasonable amount of information to the end user will not harm them.  There are times when it will provide a real benefit.